### ISSUE TITLE
Add tests for manipulated block data

### ISSUE THREAD

**Issue by webmaster128** [LiskHQ/lisk#1871]
### Expected behavior

When a single field in a block is manipulated, verifySignature must return false. This should be covered by unit tests.

### Actual behavior

The two tests of verifySignature check that

1. broken blocks throw
2. verifySignature returns true for valid blocks

These tests will also pass for an implementation that statically returns `true` for every block with the correct data type.

### Steps to reproduce

See `test/unit/logic/block.js` in current development branch

### Which version(s) does this affect? (Environment, OS, etc...)

* development

**Comment by webmaster128** [comment]
Oh the "Closes #XYZ" command of Github does not work in titles.

> For example, to close an issue numbered 123, you could use the phrases "Closes #123" in your pull request description or commit message. Once the branch is merged into the default branch, the issue will close.

https://help.github.com/articles/closing-issues-using-keywords/
